### DISCUSSION TITLE
Your two cents to help prest project

### DISCUSSION THREAD

**Discussion by arxdsilva** [prest/prest#732] [Ideas]
Hello community!

We're very happy to see that we're growing steadly and that we've gathered many great people here. Now we'd like to ask for your help, in order to create better products for prest, we'd like to know your use case and what you'd need improved or added.

Feel free to send here in the discussion or send me an email to opensource@prestd.com . This step will shape our future as a project and your help is much needed!

1. Are you or your company using prest in production?
2. How are you using it?
3. How many instances of prest do you have?
4. Is there any other product that you'd like to have with prest?
5. Would you or your company use a prest cloud (managed or not) in the future?

Best,
Arthur Silva

**Comment by vmesel** [comment]
1) Yes, we intend to do. We are still in the prototyping phase of the architecture

2) Right now it's just going to be used as a headless BI for some financial data (can't share all of the structure, but it's going to be used in the future).

3) 1 on a t3.medium on AWS - US West 2

4) There are some things: There is a cube.js feature that is nice: the playground: it allows you to play with all of the schemas you have available on the database and see the results as charts or tables); The other feature that I would really appreciate is pREST having its own auth database, allowing it to be partitioned from the data-database and handling all of the auth on this other db,
